ENGAGEHF/Dashboard/Notifications/Views/NotificationRow.swift

```diff
@@ -0,0 +1,111 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SpeziViews
+import SwiftUI
+
+
+struct NotificationRow: View {
+    @Binding var notification: Notification
+    @State private var isExpanded = false
+    
+    
+    var body: some View {
+        VStack(alignment: .leading) {
+            HStack(alignment: .center) {
+                Text(notification.type)
+                    .font(.subheadline)
+                    .foregroundStyle(.secondary)
```

Comment:
Looking at the text in the Health App, we should make this a bit smaller, all uppercase, and a smaller padding to the top and bottom.